app/controllers/alerts_list_controller.rb

```diff
@@ -16,6 +16,14 @@ def index
     redirect_to :action => 'show'
   end
 
+  def class_icons
+    res = {}
+    ['ManageIQ::Providers::Kubernetes::ContainerManager::ContainerNode'].each do |klass|
```

Comment:
Wait, is this really always just this one class?

Or are we expecting more to be added?